cmake/recipes/data.cmake

```diff
@@ -0,0 +1,19 @@
+# data
+# License: MIT
+
+message(STATUS "Third-party: fetching 'meshes'")
+
+include(FetchContent)
+FetchContent_Declare(
```

Comment:
I'd suggest to use `ExternalProject` for data so it can be downloaded at make time (in parallel). This will make cmake configure faster. See example:
https://github.com/adobe/lagrange/blob/aeebd56eaaf9d3ca448174501781c8bedaa05b63/cmake/lagrange/lagrange_download_data.cmake#L27